src/Themes.scss

```diff
@@ -296,8 +296,10 @@ body.light-theme {
     }
 
     button.btn.ShareButton {
-        background-color: #F1F2F4;
         color: black;
+        &:not(:hover) {
```

Comment:
Setting more specific rule for the base background-color seems not a good approach. We could have less specific rule for the ShareButton, so hover style can be applied from ShareButton component -- removing button.btn in front of the .ShareButton and setting background-color how it was before would fix this issue.

Reply:
```
    .ShareButton {
        color: black;
        background-color: #F1F2F4;
    }
```

Reply:
I fixed this instance of it, but we still have lots of examples from the earlier theme work.  We can discuss a wider CSS cleanup in another ticket, or do it incrementally as we improve other elements.